pages/_app.tsx

```diff
@@ -0,0 +1,9 @@
+import '../styles/globals.scss'
```

Comment:
use absolute path's instead of relative

Reply:
How to know when to copy relative paths or absolute paths?
Or I should always copy absolute paths

Reply:
You can read about the pros and cons of absolute and relative imports here: [link]( https://beqode.com/blog/absolute-vs-relative-import-paths-nodejs)